src/styles/index.scss

```diff
@@ -1,3 +1,79 @@
+$days-count: 31;
+$day-size: 100px;
+$day-gap: 1px;
+$day-color: #eee;
+$border: 1px solid black;
+$calendar-columns: 7;
+$font-size: 30px;
+$day-font-family: Arial, Helvetica, sans-serif;
+
 body {
   margin: 0;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 100vh;
+}
+
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  width: $calendar-columns * $day-size + ($calendar-columns - 1) + $day-gap +
+    $calendar-columns * 2;
```

Comment:
The calculation of the width seems to be incorrect. According to the [Code Style Rules ❗️](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules), the width should be calculated based on the size of a day and the gaps between days. Make sure to account for the number of gaps, which is one less than the number of columns, and that the total width accommodates all days and gaps correctly.